GX-TimeFrequency/src/minimax.F90

```diff
@@ -115,8 +123,8 @@ subroutine print_minimax(this)
 
     write(*, *) 'Minimax Grid'
     write(*, *) '# Grid point, weight:'
```

Comment:
This needs updating to say: `#Frequency grid point, freq weight, time grid point, time weight`